generic-helpers/pom.xml

```diff
@@ -75,6 +75,22 @@
 		    <artifactId>jasypt</artifactId>
 		    <version>1.9.3</version>
 		</dependency>
+		<dependency>
+			<groupId>redis.clients</groupId>
+			<artifactId>jedis</artifactId>
+			<exclusions>
+			<exclusion>
+				<groupId>org.json</groupId>
+				<artifactId>json</artifactId>
+			</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>it.ozimov</groupId>
+			<artifactId>embedded-redis</artifactId>
+			<version>0.7.2</version>
```

Comment:
This is not latest version of embaedded-redis, 
https://mvnrepository.com/artifact/it.ozimov/embedded-redis/0.7.3

Also if it have Critical/High vulnerability, it need to be resolved before merging the PR.
If we have medium severity vulnerability Please log a ticket as per **Elimu's Security Bug Fix Policy** available in confluence document.